packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -584,3 +583,268 @@ test(`zcf.makeZCFMint - burnLosses - seat exited`, async t => {
+  const expectedMethods = [
+    'exit',
+    'kickOut',
+    'getNotifier',
+    'hasExited',
+    'getProposal',
+    'getAmountAllocated',
+    'getCurrentAllocation',
+    'isOfferSafe',
+    'stage',
+  ];
+  const { zcf } = await setupZCFTest();
+  const makeZCFSeat = () => zcf.makeEmptySeatKit().zcfSeat;
+  const seat = makeZCFSeat();
+  t.deepEqual(Object.keys(seat), expectedMethods);
+});
+
+test(`zcfSeat.getProposal from zcf.makeEmptySeatKit`, async t => {
+  const { zcf } = await setupZCFTest();
+  const makeZCFSeat = () => zcf.makeEmptySeatKit().zcfSeat;
+  const seat = makeZCFSeat();
+  t.deepEqual(seat.getProposal(), {
+    exit: {
+      onDemand: null,
+    },
+    give: {},
+    want: {},
+  });
+});
+
+test(`zcfSeat.hasExited, exit from zcf.makeEmptySeatKit`, async t => {
+  const { zcf } = await setupZCFTest();
+  const { zcfSeat, userSeat } = zcf.makeEmptySeatKit();
+  t.falsy(zcfSeat.hasExited());
+  zcfSeat.exit();
+  t.truthy(zcfSeat.hasExited());
+  t.deepEqual(await E(userSeat).getPayouts(), {});
+});
+
+test(`zcfSeat.hasExited, kickOut from zcf.makeEmptySeatKit`, async t => {
+  const { zcf } = await setupZCFTest();
+  const { zcfSeat, userSeat } = zcf.makeEmptySeatKit();
+  t.falsy(zcfSeat.hasExited());
+  const msg = `this is the error message`;
+  const err = zcfSeat.kickOut(Error(msg));
+  t.is(err.message, msg);
+  t.truthy(zcfSeat.hasExited());
+  t.deepEqual(await E(userSeat).getPayouts(), {});
+});
+
+test(`zcfSeat.isOfferSafe from zcf.makeEmptySeatKit`, async t => {
+  const { zcf } = await setupZCFTest();
+  const { moola } = setup();
+  const { zcfSeat } = zcf.makeEmptySeatKit();
+  // Anything is offer safe with no want or give
+  // @ts-ignore
+  t.truthy(zcfSeat.isOfferSafe());
+  t.truthy(zcfSeat.isOfferSafe({ Moola: moola(0) }));
+  t.truthy(zcfSeat.isOfferSafe({ Moola: moola(10) }));
+});
+
+const allocateEasy = async (
+  zcf,
+  zcfMintKeyword,
+  zcfSeat,
+  gainsKeyword,
+  gainsValue,
+) => {
+  // Mint some gains to change the allocation.
+  const zcfMint = await zcf.makeZCFMint(zcfMintKeyword);
+  const { amountMath } = zcfMint.getIssuerRecord();
+  zcfMint.mintGains({ [gainsKeyword]: amountMath.make(gainsValue) }, zcfSeat);
+  return zcfMint.getIssuerRecord();
+};
+
+test(`zcfSeat.getNotifier`, async t => {
+  const { zcf } = await setupZCFTest();
+  const { zcfSeat, userSeat } = zcf.makeEmptySeatKit();
+  const notifier = zcfSeat.getNotifier();
+  // These are different notifiers
+  t.not(notifier, await E(userSeat).getNotifier());
+
+  // Mint some gains to change the allocation.
+  const { brand: brand1 } = await allocateEasy(zcf, 'Stuff', zcfSeat, 'A', 3);
+  t.deepEqual(await notifier.getUpdateSince(), {
+    updateCount: 2,
+    value: {
+      A: {
+        brand: brand1,
+        value: 3,
+      },
+    },
+  });
+
+  const { brand: brand2 } = await allocateEasy(zcf, 'Stuff2', zcfSeat, 'B', 6);
+  t.deepEqual(await notifier.getUpdateSince(), {
```

Comment:
```suggestion
  t.deepEqual(await notifier.getUpdateSince(2), {
```
The second call wants updates since the first call.

Reply:
Great, thanks. If I didn't put in an updateCount, I thought it got the latest, which would be the same. Is that not true?